coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/PersistentDatabase.java

```diff
@@ -0,0 +1,120 @@
+package org.coralibre.android.sdk.internal.database;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.room.Room;
+
+import org.coralibre.android.sdk.internal.crypto.ppcp.CryptoModule;
+import org.coralibre.android.sdk.internal.crypto.ppcp.ENNumber;
+import org.coralibre.android.sdk.internal.crypto.ppcp.TemporaryExposureKey;
+import org.coralibre.android.sdk.internal.database.model.CapturedData;
+import org.coralibre.android.sdk.internal.database.model.GeneratedTEK;
+import org.coralibre.android.sdk.internal.database.model.IntervalOfCapturedData;
+import org.coralibre.android.sdk.internal.database.model.IntervalOfCapturedDataImpl;
+import org.coralibre.android.sdk.internal.database.model.entity.EntityCapturedData;
+import org.coralibre.android.sdk.internal.database.model.entity.EntityGeneratedTEK;
+import org.coralibre.android.sdk.internal.database.persistenDatabase.RoomDatabaseAbstrImpl;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class PersistentDatabase implements Database {
```

Comment:
From what I understand from [`provideDiagnosisKey()`'s documentation](https://developers.google.com/android/exposure-notifications/exposure-notifications-api#providediagnosiskeys), shouldn't the database also provide a way to store diagnosis keys?

Reply:
diagnosies keys are the SKs ... just concat 14 of them in a row and you have a diagnosis key... correct me if im wrong (propably).

Reply:
I am probably blind, but I do not understand what should be called to store diagnosis keys in the database

Reply:
This: https://github.com/CoraLibre/CoraLibre-android-sdk/pull/31/files/d262691f90aa8206cb715a079d4c88477fba288b#diff-1a28eeb4b08ecf636ff52b28ff3f6999R52

Reply:
I just kept the existing interface of the Database, since my goal here was to make the current version work with a real persistent db before anything else. Changes to the interface might make sense, but I'd suggest to create own issues / PRs for such.